README.md

```diff
@@ -15,7 +15,8 @@
   <div>
     <img src="contrib/warp.png" width="230" alt="Warp" />
   </div>
-  <b>Warp is a modern, Rust-based terminal with AI built in so you and your team can build great software, faster.</b>
+  <b>Warp is a modern, Rust-based terminal with AI built in so you
+    and your team can build great software, faster.</b>
```

Comment:
This is not releated to the PR, right?

Reply:
Yes, modify this just because some error in CI